analytics/src/analytics_ios.mm

```diff
@@ -231,6 +231,69 @@ void LogEvent(const char* name) {
   [FIRAnalytics logEventWithName:@(name) parameters:@{}];
 }
 
+// Declared here so that it can be used, defined below.
+NSDictionary* MapToDictionary(const std::map<Variant, Variant>& map);
+
+// Converts the given vector into an ObjC NSArray of ObjC objects.
+NSArray* VectorToArray(const std::vector<Variant>& vector) {
```

Comment:
This (and the Android version above) seem slightly misnamed since they are specifically for converting a vector of *maps* to an array

Reply:
Fair, renamed to VectorOfMapsToArray.  Originally on iOS I converted everything, but on Android that isn't as feasible with how the ArrayList works, and then looking over the parameters the only times Arrays are used are with Dictionaries, so went with that.